sros2/sros2/policy/schemas/policy.xsd

```diff
@@ -10,14 +10,14 @@
     <xs:element name="policy" type="Policy" />
     <xs:complexType name="Policy">
         <xs:sequence minOccurs="1" maxOccurs="1">
-            <xs:element name="contexts" type="Contexts" />
+            <xs:element name="enclaves" type="Contexts" />
```

Comment:
You can compare the .xsd from the design PR to check. Not sure this repo will need to mention an kind of `context` string afterward.

Reply:
This non-change was on purpose in the sense that the resulting xsd is valid and usable without additional changes. The idea was the it will be overwritten by whatever the final version of the design doc is (including new tags and tag changes). But I can copy paste it now if we prefer

Reply:
copied xsd from PR in https://github.com/ros2/sros2/pull/198/commits/2ae51fd754c5d9d800db81ac8de110e4e76e85c2